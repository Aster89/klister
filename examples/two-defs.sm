#lang kernel

[define id [lambda [x] x]]

[define five [quote id]]

[example [id five]]
