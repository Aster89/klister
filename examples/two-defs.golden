#[/Users/gelisam/working/haskell/klister/examples/two-defs.kl:5.21-5.23]<id> : Syntax
